osu.Game/Overlays/WikiOverlay.cs

```diff
@@ -38,11 +39,20 @@ public partial class WikiOverlay : OnlineOverlay<WikiHeader>
 
         private WikiArticlePage articlePage;
 
+        private Bindable<Language> language;
+
         public WikiOverlay()
             : base(OverlayColourScheme.Orange, false)
         {
         }
 
+        [BackgroundDependencyLoader]
+        private void load(OsuGameBase game)
+        {
+            // Fetch current language on load for translated pages (if possible)
+            language = game.CurrentLanguage.GetBoundCopy();
```

Comment:
I think this should have a value changed binding (in `LoadComplete`) to reload the current page when the language changes.